.travis.yml

```diff
@@ -69,7 +69,7 @@ install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       OS_PACKAGES=libffi ;
     fi ;
-    conda create -q -n test-environment python="$TRAVIS_PYTHON_VERSION" pip redis notebook bokeh ruamel_yaml anaconda-client requests psutil $OS_PACKAGES
+    conda create -q -n test-environment python="$TRAVIS_PYTHON_VERSION" pip redis notebook bokeh ruamel_yaml anaconda-client requests psutil mock $OS_PACKAGES
```

Comment:
we can do that... (If I didn't overlooked things on the CI files, we need to check them since I think there is some conditions/logic there to be considered)